src/SingleProject/Resizetizer/src/Resizetizer.csproj

```diff
@@ -18,6 +18,10 @@
     <EmbeddedResource Include="Resources\**" LogicalName="%(RecursiveDir)%(Filename)%(Extension)" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="StrongNamer" />
```

Comment:
Why use this package instead of just adding an SNK file and doing signing the regular way?

Reply:
This signs dependencies - not the main dll. But I don't think we need it for resizetizer as it has no unsigned deps.

Reply:
What other DLLs would it be signing? From a 3rd party?

I'm a bit concerned if we'd be adding a strong name to some other DLLs that aren't even ours. That strikes me as odd... I understand the core requirement of needing things signed, but signing someone else's DLLs is unusual.

What would happen if we don't sign the dependencies and only sign our own DLLs? Is that even possible, and if so, would it work if we did?

(We do _Authenticode_ sign 3rd party DLLs, but that's strictly additive, and has no other effect on the DLL.)

Reply:
If you're strong naming (which is different than authenticode signing), you can't reference dependencies which are _not_ strong named.  @dsplaisted 's StrongNamer helps by 'fixing' dependencies to also have a strong name as part of the build.

In any case, as Matthew mentioned, we don't think we actually need to do this since as luck would have it, our dependencies already are strong named :)

Reply:
When we sign, we are rather "strong naming" - basically giving the dll an ID so the runtime knows if it matches another DLL. In our case, the filename is not enough because we have the same name for net7 and net8. This tool rather forces a ID of a name, version and a "key" which is just a number checked into the repo.

However, in this case, we are no longer doing this and just merging all the MAUI dlls and giving it a special name.